php/PhpParser.g4

```diff
@@ -673,8 +705,9 @@ functionCallName
     : qualifiedNamespaceName
     | classConstant
     | chainBase
+    | parenthesis // TODO: Hack for IIFE, seen in generatorReturn.php
```

Comment:
Maybe `parentheses` (plural form)?

Reply:
Needs more invesgation, will do it after I finish \u escape syntax.

Reply:
Got it. I'll do it tomorrow.